CMakeLists.txt

```diff
@@ -34,7 +34,13 @@ find_package(LZMA REQUIRED)
 find_package(OpenSSL REQUIRED)
 find_package(Sqlite3 REQUIRED)
 find_package(ZLIB REQUIRED)
-
+find_package(PkgConfig REQUIRED)
+find_library(ZCHUNKLIB NAMES zck)
```

Comment:
Could you write a `FindZChunk.cmake` similar to the `FindGLIB2.cmake` module and use that instead?

Reply:
Ok, let me know if this works or you want something different